timbermill-python/timbermill/timberlog_event_handler.py

```diff
@@ -81,6 +85,7 @@ def create_event(event_type: str, text: dict, name: str = None, task_id: str = N
     event_strings = __dict_values_to_str(event_strings)
 
     event = {'@type': event_type, consts.TASK_ID: task_id, 'context': context, 'strings': event_strings, 'metrics': metrics, 'text': text, 'time': event_time, 'env': ENV}
+    event = {**STATIC_EVENT_PARAMS, **event}
```

Comment:
@shimk52 Can you put it only on start events?

Reply:
done